docs/docs/behaviors/index.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Behavior Overview
+sidebar_label: Overview
+---
+
+# Behavior Overview
+
+## Key press behavior
+
+| Binding       | Behavior                                     | Description                                                                                                                                                                                                                                       |
+| ------------- | -------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `&kp`         | [Key Press](key-press.md)                    | Send keycodes to the connected host when a key is pressed                                                                                                                                                                                         |
+| `&trans`      | [Transparent Key Press](misc.md#transparent) | Pass the keypress down to the next active layer in the stack for processing                                                                                                                                                                       |
+| `&none`       | [None Key Press](misc.md#none)               | Swallows and stops the keypress, no keycode will be sent nor will the keypress be passed down to the next active layer in the stack                                                                                                               |
```

Comment:
I'm not sure if placing them under "miscellaneous" is appropriate. 

Initially torn about it, I ultimately included them alongside other behaviors related to key presses, as they significantly influence key press handling, especially across multiple active layers. 

I find, having them in the miscellaneous section might downplay their significance and hinder comprehension of how key presses are handled if multiple layers are active.

Reply:
While you are right about the last paragraph, I really don't think they fit in the key press behaviors because fundamentally they aren't related to sending keycodes to the computer. The bindings that transparent key falls into might as well be a layer change key or a BT binding, not necessarily a key press.

Maybe we can find a better name than "miscellaneous" for it, and place it right after this section so they aren't overlooked.

Reply:
I agree that they don't really fit in. I moved it into a new section below key press behaviors and called it `Miscellaneous` given that's currently also the name of the behavior page itself containing `&trans` and `&none`  in the navigation tree. 

Therefore, unless that is also to change we should stick with that name for this section to avoid confusion.